package.json

```diff
@@ -3,8 +3,7 @@
   "private": true,
   "scripts": {
     "bootstrap": "yarn && yarn conda:forge && yarn dependencies && yarn nbstripout:install && yarn install:dev",
```

Comment:
could add a `yarn git:clean` here?

Reply:
Ooo, don't think so... hmmm... that one I think we need to leave up to people actively deciding to do it. `git clean` can be a little nuclear... 

If they have just cloned the repo, `git clean` won't be needed. If they have things left over in the repo and it is causing issues, I think it needs to be a choice to do it.

Maybe though we could have a bat script called "start fresh" that does the following:

```batch
call conda deactivate
yarn conda:delete
yarn git:clean
start ./bootstrap-with-conda-env.bat
```